src/cli_keystore/cli_keystore.js

```diff
@@ -42,6 +41,21 @@ program
   .version(pkg.version)
   .description('A tool for managing settings stored in the Kibana keystore');
 
+function getKeystore() {
```

Comment:
This function is pretty important - can we add test coverage there?

Reply:
3c36f55